### ISSUE TITLE
Accessibility issue for Datagrid > Placeholder text.

### ISSUE THREAD

**Issue by Shivaraj-Patil** [vmware/clarity#4896]
The placeholder text for datagrid is rendered in text color is #B3B3B3, background color is #FFFFFF, color contrast ratio is less than 2.84:1. This is violating the WCAG 2.1 -> 1.4.3 Contrast (Minimum) (Level AA).

**Comment by gnomeontherun** [comment]
Clarity cares deeply about accessibility, and we work with our accessibility team to identify and triage issues. We have an internal tracking system to identify and verify accessibility issues, which has captured this already. Some issues with Clarity Angular are difficult to solve without causing side effects or breaking changes, and we have to balance concerns with fixing issues and breaking changes. As we build out Clarity Core, we are able to address these root cause accessibility issues and recommend adopting the Clarity Core implementations as they become available. We are closing this issue as we have the issues tracked internally. Please follow our development and releases to see when we release relevant Clarity Core components to make this possible.
